### ISSUE TITLE
Filter Response Normalization Layer: Eliminating Batch Dependence in the Training of Deep Neural Networks (CVPR2020)

### ISSUE THREAD

**Issue by iwa-shi** [iwa-shi/paper_memo#113]
Link: http://openaccess.thecvf.com/content_CVPR_2020/html/Singh_Filter_Response_Normalization_Layer_Eliminating_Batch_Dependence_in_the_Training_CVPR_2020_paper.html

## 概要
Batch Normalizationの改良版FRNを提案

## 詳細
- BNを使うと、性能はバッチサイズに大きく依存する。これを解決するためにGroup Normなどの手法が提案されてきたが、結局バッチサイズが大きいときはBNより性能が下がる。（画像１）
- 通常使われるBN+ReLUは画像２のように表されるが、提案手法FRNでは画像３のようにする。平均を引いてセンタリングしない、分母のεを学習パラメータにする、ReLUの代わりに学習パラメータτを使うTLUを使う、などが変更点
- 結果は画像４のように全バッチサイズでBNを上回った

<img src="https://github.com/iwa-shi/paper_memo/blob/master/assets/figures/44-1.jpg?raw=true" width="80%"/> 
<img src="https://github.com/iwa-shi/paper_memo/blob/master/assets/figures/44-2.jpg?raw=true" width="80%"/> 
<img src="https://github.com/iwa-shi/paper_memo/blob/master/assets/figures/44-3.jpg?raw=true" width="80%"/> 
<img src="https://github.com/iwa-shi/paper_memo/blob/master/assets/figures/44-4.jpg?raw=true" width="80%"/>
